core/lib/workarea/latest_version.rb

```diff
@@ -0,0 +1,25 @@
+module Workarea
+  module LatestVersion
+    def self.get
+      Rails.cache.fetch('workarea/version/latest', expires_in: 3.days) do
+        ENV['HTTP_PROXY'] || ENV['HTTPS_PROXY']
```

Comment:
Is this just leftover cruft? `Net::HTTP` will pay attention to these, so I believe it isn't necessary to add them to the request.

Reply:
I based this on https://github.com/workarea-commerce/workarea/blob/master/core/lib/workarea/ping_home_base.rb#L9 and assumed there was some wizardry behind invoking the variables beforehand. If its cruft in both places I can remove

Reply:
It's gotta be cruft, go for it.